app/models/purl.rb

```diff
@@ -47,6 +47,11 @@ def self.with_filter(filtering_params)
     results
   end
 
+  # Sends a message to the indexer_topic, which will cause this object to be reindexed
+  def produce_indexer_log_message
+    Racecar.produce_sync(value: as_public_json.to_json, topic: Settings.indexer_topic, key: druid)
```

Comment:
Back in the initial implementation, you tried to use `produce_sync` and removed it:

https://github.com/sul-dlss/purl-fetcher/pull/582/commits/664619592a3e2156d5ad69aa18016e5f02048cbd

The history is unclear, but do you remember what problem that fixed?

Reply:
I was thinking that was a more idiomatic usage of Racecar.